.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,3 +23,9 @@ for inclusion in changelog.
 #### Licensing
 
 This contribution is under the terms of the Apache 2.0 License: Yes
+
+#### Before merging
+<!-- Run integration and end-to-end tests before merging -->
+- [ ] cluster-state-service end-to-end tests pass. Required setup details in cluster-state-service/internal/Readme.md (`cd cluster-state-service; make e2e-tests; cd ../`)
```

Comment:
I don't want to duplicate the giant setup section from Readme. I think it's enough to redirect them to Readme